lldb/source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -272,6 +272,7 @@ void SwiftREPL::Initialize() {
   swift.Insert(lldb::eLanguageTypeSwift);
   PluginManager::RegisterPlugin(ConstString("swift"), "The Swift REPL",
                                 &CreateInstance, swift);
+  ModuleList::GetGlobalModuleListProperties().SetUseSwiftDWARFImporter(false);
```

Comment:
I'll defer this to Adrian, but it would be great if we can add a comment here on why we're disabling the DWARF importer in the REPL.

Reply:
Agreed. There should be a comment explaining that the DWARFImporter never fails and that the REPL doesn't use DWARF type info, etc...

Reply:
Done! Please let me know if the message is no good